neofetch

```diff
@@ -2358,7 +2507,7 @@ get_song() {
 
     printf -v players "|%s" "${players[@]}"
     player="$(ps aux | awk -v pattern="(${players:1})" \
-                '!/ awk / && !/iTunesHelper/ && match($0,pattern){print substr($0,RSTART,RLENGTH); exit}')"
```

Comment:
Looks like this branch needs a rebase with master.

Reply:
I'm sorry about this, but I'm not terribly experienced with git (especially working on a larger project like this) and I can't for the life of me get this branch to rebase properly. I've tried a couple different methods so far (most recently [this](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request)) and it still doesn't seem to apply the changes from these more recent commits, though it definitely seems to go through and says it's up to date with the remote master.

Am I just missing something simple? It's driving me crazy.